report/clients/standalone.js

```diff
@@ -7,7 +7,8 @@
 
 /* global document window ga DOM ReportRenderer ReportUIFeatures Logger */
 
-(function __initLighthouseReport__() {
+// eslint-disable-next-line no-unused-vars
+function __initLighthouseReport__() {
```

Comment:
was this a mistake that this is back? or was this change in another PR too? I'm losing track here.

Reply:
Merge mistake.

Reply:
Not a mistake... same change as #12765 just hasn't been merged to master yet.